tier1/{{cookiecutter.project_slug}}/repolinter.json

```diff
@@ -47,10 +47,10 @@
         "rule": {
           "type": "file-existence",
           "options": {
-            "globsAny": [
-              "{docs/,.github/,}CONTRIBUTING.md"
-            ],
-            "nocase": true
+            "globsAny": ["{docs/,.github/,}CONTRIBUTING.md"],
+            "nocase": true,
+            "file-name": "CONTRIBUTING.md",
+            "file-content": "# How to Contribute \n <!-- Basic instructions about where to send patches, check out source code, and get development support.--> \n We're so thankful you're considering contributing to an [open source project of the U.S. government](https://code.gov/)! If you're unsure about anything, just ask -- or submit the issue or pull request anyway. The worst that can happen is you'll be politely asked to change something. We appreciate all friendly contributions. \n We encourage you to read this project's CONTRIBUTING policy (you are here), its [LICENSE](LICENSE.md), and its [README](README.md). \n"
```

Comment:
Instead of the `How to Contribute` section, consider changing `file-content` to a comment giving a short explanation on CONTRIBUTING.md since the `How to Contribute` check already includes the same content here: https://github.com/DSACMS/repo-scaffolder/pull/214/files#diff-cca7916ac0efa284744a1fe22c18525bc7df68f3ef157d83d0ba929eb74dbbcbR459